apps/pie-docs/src/components/thumbnail/overview/overview.md

```diff
@@ -0,0 +1,307 @@
+      "**Image:** The fill can be an image, placeholder icon or a fill color."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+Use it for logos or as previews for images that lead to primary content.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/variant-default.svg",
+    alt: "A thumbnail component with a centred, small transparent image preview.",
+    width: "64px",
+    variant: 'secondary'
+} %}
+
+### Outline
+
+A border can be applied to ensure the container of the thumbnail is visible.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/variant-outline.svg",
+    alt: "A thumbnail placeholder with a thin border and a centred, small transparent image preview.",
+    width: "64px",
+    variant: "secondary"
+} %}
+
+---
+
+## Modifiers
+
+### Padding
+
+Adds extra space around the thumbnail, useful to center and add space around logos.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/modifier-padding.svg",
+    alt: "A thumbnail placeholder with additional space around an image preview.",
+    width: "64px",
+    variant: "secondary"
+} %}
+
+### Container fill
+
+The fill behind the image is set to $container-default by default, it can be changed as needed to any container colour token.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/container-fill.svg",
+    alt: "A thumbnail placeholder with a filled background behind image preview.",
+    width: "64px",
+    variant: "secondary"
+} %}
+
+---
+
+## Sizes
+
+Thumbnails have 3 main sizes to account for corner radius readability,XSmall, Small and Medium, each of them with maximum and minimum widths and heights to signal where it’s necessary to change into the variant with a different corner radius.
+
+The following table outlines the available Thumbnail sizes:
+
+{% notification {
+  type: "information",
+  message: "To calculate the border-radius of the inner container, subtract the padding (2px) from the thumbnail radius."
+} %}
+
+{% componentDetailsTable {
+  tableData: sizes
+} %}
+
+
+{% usage {
+    do: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/thumbnail/sizes-example-do.svg",
+            width: "128px",
+            alt: "An example of a correctly used thumbnail displaying a burger meal. The image is clear, properly sized, and fits within the designated thumbnail area."
+        }]
+    },
+    dont: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/thumbnail/sizes-example-dont.svg",
+            width: "128px",
+            alt: "An example of incorrect thumbnail usage, displaying an illustration. This is discouraged because thumbnails should contain real images, not illustrations."
+        }]
+    }
+} %}
+
+### Aspect ratio
+
+The Thumbnail component can be easily resized along the aspect ratio constraints and always in 8px increments.
```

Comment:
The text here is also a bit confusing – I'd assume the 8px increments is more related to the `size` prop, but it's mentioned here.

Reply:
moved to the sizes section